### DISCUSSION TITLE
冗長表現の対応方針

### DISCUSSION THREAD

**Discussion by KenichiroTanaka** [pgsql-jp/jpug-doc#2631] [General]
https://github.com/pgsql-jp/jpug-doc/pull/2625
にいただいている冗長表現（「・・・することができる」→「・・・できる」）ですが、翻訳時点でどこまで
強制すべきかご意見をいただけないでしょうか。
進め方としては以下があると思います。

①PR時点でチェックし、修正を依頼する
②PR時点では特にチェックせず、どこか別のタイミングで修正する
③他にあれば

①で進めるのであれば冗長表現は禁止するなどのルール決めどこかに掲載する必要があると思います。
②の場合は別のタイミングを決める必要があるかと思います。

皆様のご意見をいただけないでしょうか。

**Comment by tatsuo-ishii** [comment]
(1)に賛成です。ただ、一律禁止などの単純なルールを決めることはできず、文脈などの判断が必要になるので、ルールを決めるとしても

「不必要な冗長表現（「・・・することができる」）は避ける。

のようなやや曖昧なルールにならざるを得ないと思います。

https://lifelikewriter.com/verbose-sentences/

**Comment by noborus** [comment]
2で良いと思います。
新しいバージョンの翻訳のときに追加分であればチェックしても良いですが、
一部の変更は近くの行に対しても修正、レビューをするようにしても余計に時間がかかります。
時間を掛けても他の箇所では残るため別途修正することになります。
同じような変更はまとめてした方が修正もレビューもしやすくなります。
コードの場合は意図の違う変更は別で出すのが良いとされていて、ドキュメントもなるべくそれに沿ったほうが良いと思います。

**Comment by tatsuo-ishii** [comment]
田中さん
> ②PR時点では特にチェックせず、どこか別のタイミングで修正する
> ①で進めるのであれば冗長表現は禁止するなどのルール決めどこかに掲載する必要があると思います。

これはつまり、(2)の場合はルールは決めないということになるのでしょうか？もしそうなら、ルールを決める必要がない理由を教えてください。

**Reply by KenichiroTanaka** [reply]
確かに、ルールを決めない、ということは無いですね。
「①であれば〜」の文言は撤回させてください。

**Comment by koizumistr** [comment]
> 「・・・することができる」→「・・・できる」

というのは単純に機械的にはできない（AIならできる？）ので、.ng.list に追加するのは無理ではないでしょうか
と書こうと思ったのですが、そういう方向の議論ではなかったのですね。失礼しました。

**Reply by koizumistr** [reply]
新しいバージョンの翻訳のときに追加分・修正分であればチェック
でいい気がします。

**Reply by KenichiroTanaka** [reply]
みなさま、ご意見ありがとうございました。
私の選択肢が良くなく分かりにくくなってしまいましたが、ご意見をまとめると
小泉さんにコメントいただいた通り、
「新しいバージョンの翻訳のときに追加分・修正分であればチェック」
の方針になるかと思います。（まとめきれてませんでしたらご指摘ください！）
